src/Components/SentEmails/index.jsx

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from "react"
+import EmailSummary from "../EmailSummary"
+
+function SentEmails () {
+    const [emails, setEmails] = useState([])
+
+    function getEmails() {
+        fetch('https://email-client-api.dev.io-academy.uk/emails')
```

Comment:
This fetch request is still requesting all emails, it should be going to /emails/sent